[EDU-5809] feat: new stencil readme files (#203)
aziontech/azion-samples: templates/stencil/stencil-app-boilerplate/readme.md

# Stencil App Boilerplate

> The **Stencil App Boilerplate** template deploys a basic Stencil (4.22.2) web application and creates basic Edge Applications configurations to optimize it, such as creating multiple Cache Settings, enabling the use of Gzip compression, etc. The template creates a GitHub repository containing your project, as well as an edge application and a domain to facilitate your access and management through Azion Edge Platform.

Reviewer: ```suggestion
The **Stencil Boilerplate** template deploys a Stencil web application and creates basic edge application configurations to optimize it, such as creating multiple cache settings, enabling Gzip compression, etc. The template also creates a GitHub repository containing your project, an edge application, and a domain to facilitate your access and management through the Azion Edge Platform.
```

Stencil is a modern compiler for building reusable, scalable web components. It allows developers to easily write standard-compliant components, providing features like JSX, TypeScript, and lazy-loading while outputting highly optimized code for fast performance.

>   This repository is an example of the new GitHub repository created during the deployment. For a more detailed step-by-step to deploy this template through Azion's platform, check the [How to deploy edge applications with the Stencil Boilerplate](https://www.azion.com/en/documentation/products/guides/stencil-boilerplate/) guide.

Reviewer: ```suggestion
This repository is an example of the new GitHub repository created during the deployment. For a more detailed step-by-step to deploy this template through Azion's platform, check the [How to deploy edge applications with the Stencil Boilerplate](https://www.azion.com/en/documentation/products/guides/stencil-boilerplate/) guide.
```

This template uses Stencil version `4.22.2`.

> 

Reviewer: ```suggestion
---
```
